examples/cube.py

```diff
@@ -222,7 +222,9 @@
     {
         "binding": 0,
         "visibility": wgpu.ShaderStage.VERTEX | wgpu.ShaderStage.FRAGMENT,
-        "buffer": {},
+        "buffer": {
```

Comment:
Just curious.  Is there a reason you added these?  In each case, you've added the default value for the option.